pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -64,11 +65,12 @@ const (
 	BasicUserRoleName       = "basic-user"
 	StatusCheckerRoleName   = "cluster-status"
 
-	BuildControllerRoleName       = "system:build-controller"
-	ReplicationControllerRoleName = "system:replication-controller"
-	DeploymentControllerRoleName  = "system:deployment-controller"
-	JobControllerRoleName         = "system:job-controller"
-	HPAControllerRoleName         = "system:hpa-controller"
+	BuildControllerRoleName            = "system:build-controller"
+	ReplicationControllerRoleName      = "system:replication-controller"
+	DeploymentControllerRoleName       = "system:deployment-controller"
+	JobControllerRoleName              = "system:job-controller"
+	HPAControllerRoleName              = "system:hpa-controller"
+	PersistentVolumeControllerRoleName = "system:pv-controller"
```

Comment:
you need to define this role in `bootstrappolicy/policy.go` (see `JobControllerRoleName` definition as an example). Include all API calls, verbs and objects the persistent volume controller's client will need to be permitted to do.